web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -362,18 +362,20 @@
               />
             {/if}
           </p>
-          <div class="flex gap-2 text-sm">
-            {#if asset.exifInfo.exifImageHeight && asset.exifInfo.exifImageWidth}
-              {#if getMegapixel(asset.exifInfo.exifImageHeight, asset.exifInfo.exifImageWidth)}
-                <p>
-                  {getMegapixel(asset.exifInfo.exifImageHeight, asset.exifInfo.exifImageWidth)} MP
-                </p>
+          {#if asset.exifInfo?.fileSizeInByte && asset.exifInfo?.exifImageHeight && asset.exifInfo?.exifImageWidth}
```

Comment:
We haven't had a condition around this block before, why did you add it now?

Reply:
Improved this further in most recent commit. The conditions now better align now and is similar to other structures in this file (check to see if exif data exists, if it does, display the block and data).

I also moved the file path to show directly below the filename when info button is clicked. A little more intuitive to see it show right next to what you are clicking on (as well as keep that code together).